packages/gatsby-theme-docs/src/components/side-by-side.js

```diff
@@ -56,11 +56,14 @@ const fullWidthStyle = css`
   max-width: unset !important;
 `;
 
-const FullWidthContainer = (props) => {
+const FullWidthContainer = ({ children, ...otherProps }) => {
   const pageData = usePageData();
   return (
-    <div css={pageData.allowWideContentLayout ? fullWidthStyle : null}>
-      {props.children}
+    <div
+      css={pageData.allowWideContentLayout ? fullWidthStyle : null}
+      {...otherProps}
+    >
+      {children}
```

Comment:
Have you seen my suggestion? The children is part of the prop, so you can just spread the entire props.